src/test/java/com/wikia/webdriver/testcases/mercurytests/old/widgettests/ApesterTests.java

```diff
@@ -0,0 +1,96 @@
+package com.wikia.webdriver.testcases.mercurytests.old.widgettests;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.TopBar;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.widget.ApesterWidgetPageObject;
+
+import org.testng.annotations.Test;
+
+@Test(groups = "Mercury_ApesterWidget")
+@Execute(onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING)
+@InBrowser(
+    browser = Browser.CHROME,
+    emulator = Emulator.GOOGLE_NEXUS_5
+)
+public class ApesterTests extends NewTestTemplate {
+
+  private TopBar topBar;
+  private Navigate navigate;
+  private ApesterWidgetPageObject widget;
+
+  private static final String APESTER_ONE_WIDGET_ARTICLE_NAME = "/wiki/ApesterMercury/OneWidget";
+  private static final String
+      APESTER_MULTIPLE_WIDGETS_ARTICLE_NAME = "/wiki/ApesterMercury/MultipleWidgets";
+  private static final String
+      APESTER_INCORRECT_WIDGET_ARTICLE_NAME = "/wiki/ApesterMercury/IncorrectWidget";
+  private static final String QUERY_1 = MercurySubpages.MAP.substring(6);
+  private static final String QUERY_2 = APESTER_ONE_WIDGET_ARTICLE_NAME.substring(6);
+
+  private void init() {
```

Comment:
What do you think about using Lombok and lazy loading instead of initializing this objects in every test?